client/my-sites/stats/stats-date-picker/index.jsx

```diff
@@ -125,20 +127,32 @@ class StatsDatePicker extends Component {
 	}
 
 	render() {
-		const { summary, translate, query, showQueryDate } = this.props;
+		const { summary, translate, query, showQueryDate, isActivity } = this.props;
 		const isSummarizeQuery = get( query, 'summarize' );
 
-		const sectionTitle = translate( 'Stats for {{period/}}', {
-			components: {
-				period: (
-					<span className="period">
-						<span className="date">{ isSummarizeQuery ? this.dateForSummarize() : this.dateForDisplay() }</span>
-					</span>
-				)
-			},
-			context: 'Stats: Main stats page heading',
-			comment: 'Example: "Stats for December 7", "Stats for December 8 - December 14", "Stats for December", "Stats for 2014"'
-		} );
+		const sectionTitle = isActivity
+			? translate( 'Activity for {{period/}}', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?